sources/platform/actors/development/programming_interface/environment_variables.md

```diff
@@ -16,59 +16,83 @@ import TabItem from '@theme/TabItem';
+| `ACTOR_MAX_PAID_DATASET_ITEMS` | For paid-per-result Actors, the user-set limit on returned results. Do not exceed this limit. |
+| `APIFY_HEADLESS` | If **1**, web browsers inside the Actor should run in headless mode (no windowing system available). |
+| `APIFY_IS_AT_HOME` | Set to **1** if the Actor is running on Apify servers. |
+| `ACTOR_MEMORY_MBYTES` | Size of memory allocated for the Actor run, in megabytes. Used to optimize memory usage. |
+| `APIFY_PROXY_PASSWORD` | The Apify Proxy password of the user who started the Actor. |
+| `APIFY_PROXY_PORT` |  |
+| `APIFY_PROXY_STATUS_URL` |  |
+| `ACTOR_STANDBY_PORT` | TCP port for the Actor to start an HTTP server to receive messages from Actor Standby. |
+| `ACTOR_STARTED_AT` | Date when the Actor was started. |
+| `ACTOR_TIMEOUT_AT` | Date when the Actor will time out. |
+| `APIFY_TOKEN` | API token of the user who started the Actor. |
+| `APIFY_USER_ID` | ID of the user who started the Actor. May differ from the Actor owner. |
+| `ACTOR_WEB_SERVER_PORT` | TCP port for the Actor to start an HTTP server to receive external messages. |
+| `ACTOR_WEB_SERVER_URL` | Unique public URL for accessing the Actor run web server from the outside world. |
+| `APIFY_API_PUBLIC_BASE_URL` | Public URL of the API, typically set to api.apify.com |
+| `APIFY_DEDICATED_CPUS` | Number of CPU cores reserved for the actor, based on allocated memory. |
+| `APIFY_DISABLE_OUTDATED_WARNING` |  |
+| `APIFY_FACT` |  |
+| `APIFY_WORKFLOW_KEY` | Identifier used for grouping related runs and API calls together. |
+| `APIFY_META_ORIGIN` |  |
+| `APIFY_SDK_LATEST_VERSION` |  |
+
 
 <!-- vale Microsoft.RangeFormat = NO -->
-Dates are always in the UTC timezone and are represented in simplified extended ISO format ([ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)), e.g. **2022-07-13T14:23:37.281Z**.
+
+:::note Date format
+
+All date-related variables use the UTC timezone and are in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format (e.g., _2022-07-13T14:23:37.281Z_).
+
+:::
 <!-- vale Microsoft.RangeFormat = YES -->
 
+## Access environment variables
+
+You can access environment variables in your code as follows:
+
 <Tabs groupId="main">
 <TabItem value="JavaScript" label="JavaScript">
 
+In Node.js, use the `process.env` object:
+
 ```js
-// To access environment variables in Node.js, use the process.env object
 console.log(process.env.APIFY_USER_ID);
 ```
 
 </TabItem>
 <TabItem value="Python" label="Python">
 
+In Python, use the `os.environ` dictionary:
+
 ```python
-# To access environment variables in Python, use the os.environ dictionary:
 import os
 print(os.environ['APIFY_USER_ID'])
 ```
 
 </TabItem>
 </Tabs>
 
-For convenience, rather than using environment vars directly, we provide a `Configuration` class
-that allows reading and updating the Actor configuration.
+## Use the `Configuration` class
+
+For more convenient access to Actor configuration, use the `Configuration` class
```

Comment:
The configuration should be a link here https://docs.apify.com/sdk/js/reference/class/Configuration